packages/flutter/lib/src/widgets/repeating_animation_builder.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+
+import 'framework.dart';
+import 'ticker_provider.dart';
+import 'transitions.dart';
+import 'value_listenable_builder.dart';
+
+/// Configures how [RepeatingAnimationBuilder] loops its animation.
+enum RepeatMode {
+  /// Each iteration starts over from the beginning once 1.0 is reached.
+  restart,
+
+  /// Each iteration runs forward, then reverses back to the beginning.
```

Comment:
Can you document whether this means the `duration` is doubled for each actual iteration, or that the curve is calculated at double speed?

(Alternatively we can document it in `RepeatingAnimationBuilder` or even in `RepeatingAnimationBuilder.duration` but I think documenting it here is more useful if this enum is to be used in other widgets.)